modules/okhttp-client/src/main/java/com/spotify/apollo/http/client/HttpClientModule.java

```diff
@@ -38,6 +39,9 @@ public static ApolloModule create() {
   protected void configure() {
     Multibinder.newSetBinder(binder(), ClientDecorator.class)
         .addBinding().toProvider(HttpClientDecoratorProvider.class);
+
+    bind(HttpClient.class).toProvider(HttpClientProvider.class);
+    bind(OkHttpClient.class).toProvider(OkHttpClientProvider.class);
```

Comment:
Binding these without the `.to...` part should make Guice find injectable constructors.